### ISSUE TITLE
Facet listed twice on public work show page

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/scihist_digicoll#623]
Under the "language" facet, the "English" facet link is shown twice on this item:
https://digital.sciencehistory.org/works/4f16c366k

**Comment by jrochkind** [comment]
Interesting, I'm guessing it's actually listed twice in the db. Not sure how that could happen. 

We could try to stop that from happening. Maybe a validation rule or (probably better) some code to automatically `uniq` the values. 

And/or we could just uniq them on display.

**Comment by eddierubeiz** [comment]
I'd be fine with the front-end solution.

(This bug isn't something that's going to be keeping me up at night, but I figured I would note it in case it's symptomatic of some larger indexing problem.)

**Comment by jrochkind** [comment]
Agreed. In this case it's probably not indexing related, if you mean Solr, because Solr is not involved in the display of that page.

**Comment by eddierubeiz** [comment]
Oh. It *is* listed twice in the DB. Well let me fix that.

**Comment by eddierubeiz** [comment]
I no longer consider this a bug. Reopen if you disagree.

**Comment by jrochkind** [comment]
I think it's a bug -- it shouldn't be possible to list twice in the db, and/or the front-end should de-dup it. 

But a pretty low-priority one. 

Thanks for noticing it and recording it @eddierubeiz !

**Comment by jrochkind** [comment]
I wonder if there should be a feature in  attr_json to use a `Set` instead of an array. But then it wouldn't be ordered, and we sometimes want it ordered. Some kind of ordered enforced-no-dups data type. Anyway, not too big a deal.

**Comment by eddierubeiz** [comment]
Another way of looking at it is as a validation bug on the work entry form.
